src/pages/ValidateLoginPage/index.js

```diff
@@ -37,17 +36,15 @@ const defaultProps = {
 };
 
 function ValidateLoginPage(props) {
-    const {canUsePasswordlessLogins} = usePermissions();
     const {preferredLocale} = useLocalize();
 
     useEffect(() => {
         const login = lodashGet(props, 'credentials.login', null);
         const accountID = lodashGet(props.route.params, 'accountID', '');
         const validateCode = lodashGet(props.route.params, 'validateCode', '');
 
-        // A fresh session will not have credentials.login and user permission betas available.
-        // In that case, we directly allow users to go through password less flow
-        if (!login || canUsePasswordlessLogins) {
+        // A fresh session will not have credentials.login available.
+        if (!login) {
```

Comment:
Changing this (and the other `if` in src/pages/ValidateLoginPage/index.website.js) resulted in [this regression](https://github.com/Expensify/App/issues/23504) where users couldn't log in using magic links. 

I should've added QA steps for testing logging in with magic links for this PR, my bad 😞